feat: implement support for end_session_endpoint (#27)
Drenso/symfony-oidc: README.md

When a user is authenticated, you will see the `REMEMBERME` cookie. You can remove the `PHPSESSID` cookie to check whether remember me is working.

### Logout

It is possible to enable "logout" through the `end_session_support` functionality of the Identity Provider, if the `end_session_endpoint` parameter is present in the .well-known endpoint it can be used. 

> Because logging out is not fully functional through OpenID because of the connected providers (for example: Azure, Facebook, etc) possibly not login the user out, this option is disabled by default.

Reviewer: ```suggestion
As logging out is fundamentally broken when using single sign-on, this option is disabled by default. This is due to the fact that logging out at the identity provider (for example: Azure, Facebook, etc) cannot guarantee the user is logged out of any other service that the user has authenticated with using the same identity provider.
```

If you want to enable the "logout" support, simply add `enable_end_session_listener: true` to your oidc listener in the firewall config. It will only work of you enabled the default Symfony `logout: true` setting in your firewall.

> By default, the listener will pass the logout `target_path` to the OpenID Provider, so the user gets redirected back to your application after login out. If you don't want this and want the user to remain at the logout confirmation page of your OpenID Provider, enable the `use_logout_target_path: false` setting.

Reviewer: ```suggestion
By default, the listener will pass the logout `target_path` to the OpenID Provider, so the user gets redirected back to your application after logging out. If you don't want this and want the user to remain at the logout confirmation page of your OpenID Provider, enable the `use_logout_target_path: false` setting.
```

> **Example: default logout path**

Reviewer: ```suggestion
**_Example: default logout path_**
```

```yaml
security:
  firewalls:
    main:
      logout: true
      oidc:
        enable_end_session_listener: true
```

> **Example: custom logout path**

Reviewer: ```suggestion
**_Example: custom logout path_**
```

```yaml
security:
  firewalls:
    main:
      logout:
        target: /my_custom_target_path
      oidc:
        enable_end_session_listener: true
```

> **Example: disable redirect to logout `target_path`**

Reviewer: ```suggestion

**_Example: disable redirect to logout `target_path`_**
```
